package.json

```diff
@@ -7,7 +7,10 @@
     "node": ">=8.0.0"
   },
   "scripts": {
-    "test": "eslint --ignore-pattern 'test/*' . && jest"
+    "test": "yarn run test:webpack4 && yan run @test:webpack5",
+    "test:webpack4": "eslint --ignore-pattern 'test/*' . && jest",
```

Comment:
`eslint --ignore-pattern 'test/*'` is not really about testing webpack 4. This should be stay in `test`